frontend/packages/topology/src/components/page/TopologyView.tsx

```diff
@@ -34,25 +39,29 @@ import TopologyFilterBar from '../../filters/TopologyFilterBar';
 import { getTopologySideBar } from '../side-bar/TopologySideBar';
 import { FilterContext } from '../../filters/FilterProvider';
 import './TopologyView.scss';
-import { useResolvedExtensions } from '@console/dynamic-plugin-sdk/src/api/useResolvedExtensions';
 
 const FILTER_ACTIVE_CLASS = 'odc-m-filter-active';
 
 interface StateProps {
-  application: string;
-  eventSourceEnabled: boolean;
+  application?: string;
+  eventSourceEnabled?: boolean;
```

Comment:
Why are these optional?

Reply:
@debsmita1 ^^

Reply:
I had to make these optional because when I am connecting DropTarget with ConnectedTopologyView it throws a type error saying I need to pass all the state & dispatch props